apierrors/errors.go

```diff
@@ -0,0 +1,36 @@
+package apierrors
```

Comment:
This package is not needed (yet)

Reply:
When doing the commits, i didn't know in which commit level I would add the apierrors so decided that I will do it straight away

Reply:
It would have made sense to add it in once you create the new handlers, but it doesn't break anything to have it in now :)